### ISSUE TITLE
Modifications on css

### ISSUE THREAD

**Issue by trisketz** [qubeshub/hubzero-cms#122]
I hope this is ok, but I made some modifications to the html and css for the homepage, community, services, and ambassadors page to fix some padding issues on the bottom. 

Also, I was not able to make the ambassadors page on QA, I keep getting an error 500.

**Comment by mdlama** [comment]
Yep, no problem on making changes on dev.  You can work there now if you would like.

I just created the Ambassadors page for you.   Also, the front page looks interesting.  :)

<img width="1423" alt="screen shot 2018-02-28 at 4 36 14 pm" src="https://user-images.githubusercontent.com/4877315/36814556-e0dadf18-1ca5-11e8-8b04-ed9cfab40659.png">

**Comment by trisketz** [comment]
Yeah, that's because the css to fix that is still on dev haha. If that css gets pushed over to QA, it'll be back to normal ;) I'm assuming someone will be able to do that at some point? All those pages are going to be weird lol.

**Comment by mdlama** [comment]
I think the easiest thing to do right now is to work on dev and make changes to css, but then test on QA by uploading the modified css files to media/css.  I went ahead and uploaded the modified css files to QA media manager, and temporarily changed the articles to point to those css files so we can see how things look on QA.  Little bit of a pain, but hopefully not too bad!
